### PR TITLE
Document global `secrets.toml` file feature

### PR DESCRIPTION
**Pull request by snehankekre**
## 📚 Context

Streamlit v1.21.0 will let users specify a global `.streamlit/secrets.toml` file, just as we allow a global `.streamlit/config.toml`.

## 🧠 Description of Changes

- Adds a "Secrets management" doc to Streamlit library's "Advanced features" section that documents how to locally develop apps with secrets

## 💥 Impact

Size:

- [ ] Small 
- [x] Not small 

## 🌐 References

- [x] [Notion](https://www.notion.so/snowflake-corp/Docs-4de336dc35074a01a3e80046f5176eb4)

**Contribution License Agreement**

By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

### CODE REVIEW

File: content/library/advanced-features/secrets-management.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+# Far more compact version!
+my_db.connect(**st.secrets.db_credentials)
+```
+
+### Error handling
+
+Here are some common errors you might encounter when using secrets management.
+
+- If a `.streamlit/config.toml` is created _while_ the app is running, the server needs to be restarted for changes to be reflected in the app.
```
Comment (sfc-gh-jcarroll) [inline content/library/advanced-features/secrets-management.md:93]: should be `secrets.toml` I believe

File: content/library/advanced-features/secrets-management.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+---
+title: Secrets management
+slug: /library/advanced-features/secrets-management
+---
+
+# Secrets management
+
+It is bad practice to store unencrypted secrets in a git repository. Suppose your application needs access to sensitive credentials. In that case, the recommended solution is to store those credentials in a file not committed to the repository and pass them as environment variables.
+
+Secrets management allows you to securely store and access secrets in your Streamlit app as environment variables.
```
Comment (sfc-gh-jcarroll) [inline content/library/advanced-features/secrets-management.md:10]: I put some suggested edits for this section in Notion

Comment (snehankekre) [reply content/library/advanced-features/secrets-management.md]: Thank you so much! Fixed on 03560c3

**Review by sfc-gh-jcarroll — APPROVED** [review]
LGTM! Thank you!!!

**Pull request merged**